modules/ossm-threescale-authentication.adoc

```diff
@@ -0,0 +1,156 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/threescale-adapter.adoc
+
+[id="ossm-threescale-authentication_{context}"]
+= Authenticating requests
+This Technology Preview release supports the following authentication methods:
+
+* *Standard API Keys*: single randomized strings or hashes acting as an identifier and a secret token.
+* *Application identifier and key pairs*: immutable identifier and mutable secret key strings.
+* *OpenID authentication method*: client ID string parsed from the JSON Web Token.
+
+== Applying authentication patterns
```

Comment:
Add explicit anchors for the unanchored headings in this module. (And all modules.)

Reply:
Done for this module. Will check other modules.